docs/UserGuide.md

```diff
@@ -426,15 +427,19 @@ The output will look like the following.
 #### Viewing all the staff(s) working a shift : `viewShift`
 
 Finds all the staff working at a particular shift. The shift can be specified either by detailing the day of the week and the time, or the day of the week and slot number.
+The result also filters the schedule GUI.
 
 * When using the -ti flag, it is in 24-hour format. Example, for 4.pm on wednesday, we use <br> `wednesday-16:00`.
 * The DAY entry is not case-sensitive.
-* If no date input is provided, the next occurrence of the provided shift is displayed.
+* If no DATE input is provided, the next occurrence of the provided shift is displayed.
+* If DATE is provided, the shift that will be looked at is the next date which satisfies the DAY entry.
 * Similarly, if only one date input is provided, the next occurrence of the provided shift *after the provided date* is displayed.
+* If no input is provided, displays output for the shift occurring now. (The current date and time)
 
 Formats:  
-`viewShift -d DAY-shift_number [da/START_DATE] [da/END_DATE]`  
-`viewShift -ti DAY-HH:mm [da/START_DATE] [da/END_DATE]`
+`viewShift -d DAY-shift_number [da/DATE]`  
+`viewShift -ti DAY-HH:mm [da/DATE]`
+`viewShift`
```

Comment:
Can we standardize to "DAYOFWEEK"? Other functions use it also, like addShift

Reply:
Yea. I think that HH:mm does not really give much information of what the field is? Any suggestions on what it should be or maybe use TIME?